modules/common/main.tf

```diff
@@ -33,7 +33,7 @@ module "ingress" {
   # inputs
   ingress_domain              = local.ingress_domain
   enable_ssh_tcp              = var.enable_ssh_tcp
-  load_balancer_access_ranges = var.whitelist_cidr
+  load_balancer_access_ranges = concat(var.whitelist_cidr, local.nat_ip_cidr)
```

Comment:
Would be good to add a comment explaining why we need the concat function here.

Reply:
Added a comment